### PR TITLE
Improve readability, and add a link to the navigator.getUserMedia API

### PR DESCRIPTION
**Pull request by enlight**
Minor improvements to the `desktopCapturer` docs to try and make it a bit easier to read.

### CODE REVIEW

File: docs/api/desktop-capturer.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # desktopCapturer
 
-> List `getUserMedia` sources for capturing audio, video, and images from a
-microphone, camera, or screen.
+> Provides information about media sources that can be used to capture audio and
```
Comment (zeke) [inline docs/api/desktop-capturer.md]: These API summaries are usually written in the imperative style. I think `Get` or `Access` would be preferable to `Provides` here.

File: docs/api/desktop-capturer.md [hunk 2]
```diff
@@ -23,29 +26,28 @@ desktopCapturer.getSources({types: ['window', 'screen']}, (error, sources) => {
...
+  console.log(e);
 }
 ```
 
-When creating a constraints object for the `navigator.webkitGetUserMedia` call,
-if you are using a source from `desktopCapturer` your `chromeMediaSource` must
-be set to `"desktop"` and your `audio` must be set to `false`.
+To capture video from a source provided by `desktopCapturer` the constraints
+passed to [`navigator.webkitGetUserMedia`] must include
+`chromeMediaSource: "desktop"`, and `audio: false`.
```
Comment (zeke) [inline docs/api/desktop-capturer.md]: A minor detail, but this should ideally be in single-quotes for consistency.

**Comment by enlight** [comment]
Made the changes suggested by @zeke

**Comment by zcbenz** [comment]
Thanks!

**Pull request merged**